test_vm/src/lib.rs

```diff
@@ -256,6 +256,24 @@ impl<'bs> VM<'bs> {
         *self.state_root.borrow()
     }
 
+    pub fn with_epoch(&self, epoch: ChainEpoch) -> Self {
```

Comment:
hmm I think this exists here already: https://github.com/filecoin-project/builtin-actors/blob/master/test_vm/src/lib.rs#L221

Reply:
:+1: